src/core/Dropdown/index.stories.tsx

```diff
@@ -65,6 +66,36 @@ MultipleSelectWithSearch.args = {
   search: true,
 };
 
+export const MultipleSelectWithSearchNoOptionsText = Template.bind({});
+MultipleSelectWithSearchNoOptionsText.args = {
+  MenuSelectProps: {
+    noOptionsText: "No results",
+  },
+  label: LABEL,
+  multiple: true,
+  onChange: noop,
+  // options: [],
+  search: true,
+};
```

Comment:
This particular example / story was not added in this PR, but I thought it'd be useful to illustrate in Storybook.  Let me know if that's not something we want to add and I can remove it.